README.md

```diff
@@ -1,31 +1,64 @@
+Cylc (pronounced silk) is a general purpose workflow engine 
+that specialises in cycling workflows and has strong scaling characteristics.
 
-**Development**: [![Contributors](https://img.shields.io/github/contributors/cylc/cylc-flow.svg?color=9cf)](https://github.com/cylc/cylc-flow/graphs/contributors) [![Commit activity](https://img.shields.io/github/commit-activity/m/cylc/cylc-flow.svg?color=yellowgreen)](https://github.com/cylc/cylc-flow/commits/master) [![Last commit](https://img.shields.io/github/last-commit/cylc/cylc-flow.svg?color=ff69b4)](https://github.com/cylc/cylc-flow/commits/master)
+Cylc was originally developed to meet the challenges of production weather
+forecasting - which is notorious for the size and complexity of its workflows.
 
-Cylc ("silk") orchestrates complex distributed suites of interdependent cycling
-(or non-cycling) tasks. It was originally designed to automate environmental
-forecasting systems at [NIWA](https://www.niwa.co.nz), however Cylc is a
-general workflow engine; it is not specialized to forecasting in any way.
+### Citations & Publications
 
-### Python 2 or Python 3 ?
+[![DOI](https://zenodo.org/badge/1836229.svg)](https://zenodo.org/badge/latestdoi/1836229)
+[![JOSS](http://joss.theoj.org/papers/10.21105/joss.00737/status.svg)](https://doi.org/10.21105/joss.00737)
+[![CISE](https://img.shields.io/website/https/ieeexplore.ieee.org/document/8675433.svg?color=orange&label=CISE&up_message=10.1109%2FMCSE.2019.2906593)](https://ieeexplore.ieee.org/document/8675433)
 
-Currently in the source code repository:
- - **master branch:** Python 3, ZeroMQ network layer, *no GUI* -  **Cylc-8 Work In Progress**
- - **7.8.x branch:** Python 2, Cherrypy network layer, PyGTK GUI - **Cylc-7 Maintenance**
+#### Cylc 7 (production)
 
-The first official Cylc-8 release (with a new web UI) is not expected until late 2019.
-Until then we recommend the latest cylc-7.8 release for production use.
+![python](https://img.shields.io/badge/python-2.6%20%7C%202.7-orange)
+
+* Production ready.
+* HTTPS network layer.
+* PyGTK GUI.
+* On the `7.8.x` branch in the source code.
+* 7.8 - Python 2.6
+* 7.9 - Python 2.7
 
 [Quick Installation](INSTALL.md) |
 [Website](https://cylc.github.io/) |
-[Documentation](https://cylc.github.io/documentation) |
-[Contributing](CONTRIBUTING.md)
+[Documentation](https://cylc.github.io/documentation)
+
+### Cylc 8 (experimental)
+
+![PyPI](https://img.shields.io/pypi/pyversions/cylc-flow.svg?color=green)
+[![PyPI](https://img.shields.io/pypi/v/cylc-flow.svg?color=yellow)](https://pypi.org/project/cylc-flow/)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/cylc-flow/badges/version.svg)](https://anaconda.org/conda-forge/cylc-flow)
+
+* Experimental alpha.
+* ZMQ (TCP) network layer.
+* In-browser GUI
+* On the `master` branch in the source code.
+
+The first official Cylc-8 release is not expected until late 2019.
```

Comment:
Should we change to "until late 2020"? Or are we considering the alpha releases as official releases? Maybe we should say when the 8.0 is released in the readme?

Reply:
Damn good point I forgot what year it is!